fdbserver/QuietDatabase.actor.cpp

```diff
@@ -611,6 +626,63 @@ ACTOR Future<Void> reconfigureAfter(Database cx,
 	return Void();
 }
 
+ACTOR Future<Void> waitForNoWrongStoreType(Database cx,
+                                           Reference<AsyncVar<ServerDBInfo> const> dbInfo,
+                                           std::string phase) {
+	state Future<bool> wrongStoreTypeCheck;
+	// Require 3 consecutive successful no wrong store type check spaced 2 second apart
+	state int numSuccesses = 0;
+
+	auto traceMessage = "QuietDatabaseWrongStoreTypeCheck" + phase + "Begin";
+	TraceEvent(traceMessage.c_str()).log();
+	loop {
+		try {
+			TraceEvent("QuietDatabaseWrongStoreTypeCheckWaitingOnDataDistributor").log();
```

Comment:
This trace event name gives me enterprise java flashbacks 💀 
I know a lot of the other trace events in QuietDatabase are similarly long, but imo it's kind of difficult to parse these event names.
We could always break it out the different phases into a separate detail field, abbreviate "QuietDatabase" as "QuietDB", etc...
We also don't have to do anything as part of this PR, but it'd be good to do eventually.

Reply:
In this case I think the long names are useful. When using grep, it allows us to prefix search which is super useful.